pkg/autoscaler/statforwarder/forwarder_test.go

```diff
@@ -126,21 +125,21 @@ func TestForwarderReconcile(t *testing.T) {
 	}
 
 	// Check the endpoints got updated.
+	el := endpoints.Lister().Endpoints(testNs)
 	if err := wait.PollImmediate(10*time.Millisecond, 2*time.Second, func() (bool, error) {
-		got, err := endpoints.Lister().Endpoints(testNs).Get(bucket1)
+		got, err := el.Get(bucket1)
 		if err != nil {
 			lastErr = err
 			return false, nil
 		}
 
-		if equality.Semantic.DeepEqual(wantSubsets, got.Subsets) {
-			return true, nil
+		if !cmp.Equal(wantSubsets, got.Subsets) {
+			lastErr = fmt.Errorf("Got Subsets = %v, want = %v", got.Subsets, wantSubsets)
+			return false, nil
 		}
-
-		lastErr = fmt.Errorf("Got Subsets = %v, want = %v", got.Subsets, wantSubsets)
-		return false, nil
+		return true, nil
 	}); err != nil {
-		t.Fatalf("Timeout to get the Endpoints: %v", lastErr)
+		t.Fatal("Timeout to get the Endpoints:", lastErr)
```

Comment:
```suggestion
		t.Fatal("Timeout to get the Endpoints: ", lastErr)
```

Reply:
`t.X()` always add a space. The only place where we always have to add a space is zap.Log calls.
Fmt needs them if both are strings, I think.